### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by dr-rodriguez**
Adding details for each table in the database.
The SpectralType table description is prior to Kelle's changes.

### CODE REVIEW

File: documentation/Instruments.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Instruments
+
+The Instruments table contains names and references for instruments referred to in other tables.
+Entries must exist in the Instruments table prior to inserting data that refer to it.
+The *name* of an instrument is required to be unique.
+
+| Column Name | Description  | Unit  | Data Type | Key Type  |
+|---|---|---|---|---|
+| name      | Unique name for the instrument |   | String(30)  | primary   |
```
Comment (Will-Cooper) [inline documentation/Instruments.md:9]: correct as is but links to Issue #99

File: documentation/ProperMotions.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# ProperMotions
+
+The ProperMotions table contains proper motion measurements in mas/yr for sources listed in the Sources table. 
+The combination of *source* and *reference* is expected to be unique.
+
+| Column Name | Description  | Unit  | Data Type | Key Type  |
+|---|---|---|---|---|
+| source        | Unique identifier for the source |   | String(100)  | primary and foreign: Sources.source   |
+| mu_ra         | Proper motion in RA | mas/yr | Float  |   |
```
Comment (Will-Cooper) [inline documentation/ProperMotions.md:9]: should we be clarifying if the quoted mu_ra is inclusive of the cos dec term?

File: documentation/Publications.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Publications
+
+The Publications table contains metadata for each publication referred to in the database. 
+Entries must exist in the Publications table prior to inserting data that refer to it.
+The *name* of a publication is required to be unique.
+
+| Column Name | Description  | Unit  | Data Type | Key Type  |
+|---|---|---|---|---|
+| name          | Unique identifier for the publication |   | String(30)  | primary   |
+| bibcode       | Bibcode entry |   | String(100)  |    |
```
Comment (Will-Cooper) [inline documentation/Publications.md:10]: bibcode or doi should be required

File: documentation/SpectralTypes.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+# SpectralTypes
+
+The SpectralTypes table contains spectral type measurements for sources listed in the Sources table. 
+The combination of *source*, *regime*, and *reference* is expected to be unique.
+Note that *regime* is a string constrained from a list of enumerated values.
+
+| Column Name | Description  | Unit  | Data Type | Key Type  |
+|---|---|---|---|---|
+| source    | Unique identifier for the source |   | String(100)  | primary and foreign: Sources.source   |
+| spectral_type | Spectral type string |  | String(10)  |   |
```
Comment (Will-Cooper) [inline documentation/SpectralTypes.md:10]: Do we want to keep spectral type unconstrained like this or require e.g. first char must be of M,L,T,Y and second char inclusively betwixt 0-9? To stop things like "sdM9" or ">M10V"?

File: documentation/Telescopes.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+# Telescopes
+
+The Telescopes table contains names and references for telescopes referred to in other tables.
+Entries must exist in the Telescopes table prior to inserting data that refer to it.
+The *name* of a telescope is required to be unique.
+
+| Column Name | Description  | Unit  | Data Type | Key Type  |
+|---|---|---|---|---|
+| name      | Unique name for the telescope |   | String(30)  | primary   |
```
Comment (Will-Cooper) [inline documentation/Telescopes.md:9]: sync with instruments as above in issue #99

**Review by Will-Cooper — APPROVED** [review]
All documentation correct as is but we may want to wait to make the underlying schema changes first as discussed in comments, then re-correct these?

File: documentation/Publications.md (hunk 3 above)
Comment (kelle) [reply documentation/Publications.md:10]: I opened issue #98 to address this. Please comment there.

File: documentation/SpectralTypes.md (hunk 4 above)
Comment (kelle) [reply documentation/SpectralTypes.md:10]: please add this comment to my PR #94, which modifies the spectral type table.

**Pull request merged**